### PR TITLE
support hhvm 4.28+

### PR DESCRIPTION
**Pull request by lexidor**

### CODE REVIEW

File: hhast-lint.json [hunk 1]
```diff
@@ -6,7 +6,6 @@
     {
       "patterns": ["*"],
       "disabledLinters": [
-        "Facebook\\HHAST\\Linters\\MustUseOverrideAttributeLinter",
```
Comment (tyronjung-quizlet) [inline hhast-lint.json:9]: 🤔

**Review by tyronjung-quizlet — APPROVED** [review]
Awesome work, thanks for doing this. 🙌

File: hhast-lint.json (hunk 1 above)
Comment (lexidor) [reply hhast-lint.json:9]: We don't need to disable linters, because we lint clean.
I can actually also remove the the DontUseReferences exception.

Comment (tyronjung-quizlet) [reply hhast-lint.json:9]: Gotcha, nice.

**Pull request merged**